hedera-node/hedera-mono-service/src/main/java/com/hedera/services/grpc/marshalling/AliasResolver.java

```diff
@@ -262,7 +267,7 @@ private void perceiveResult(
         } else if (repetitionsAreInvalid && result == Result.REPEATED_UNKNOWN_ALIAS) {
             perceivedInvalidCreations++;
         } else if (result == Result.UNKNOWN_EVM_ADDRESS) {
-            perceivedMissing++;
+            perceivedLazyCreations++;
```

Comment:
I haven't considered that the case is a potential problem, good catch!
I have implemented the fix as recommended, with a slight difference for the second point:
```
if (isSerializedProtoKey(alias)) {
    perceivedCreations++;
} else if (alias.size() == EVM_ADDRESS_SIZE) {
    perceivedLazyCreations++;
```
so that we increment the `perceivedLazyCreations` counter for the case of EVM address alias.

Also, I added a test for the secondary problem related to the counts of hollow creations if an unknown EVM address appears in both the hbar `TransferList` and a `TokenTransferList`.